conans/test/util/tools_test.py

```diff
@@ -1167,3 +1171,47 @@ def build(self):
         client.save({"conanfile.py": conanfile, "other": "hello"})
         client.run("create . user/channel", ignore_error=True)
         self.assertIn("specify a branch to checkout", client.out)
+
+
+@attr('slow')
+class RealRequestTest(unittest.TestCase):
+    """
+    Open a real server to test tools that download stuff.
+    """
+
+    thread = None
```

Comment:
class objects? Weird

Reply:
this is to make sure server starts at setUp() and it is killed in tearDown()